src/colors.js

```diff
@@ -113,25 +113,38 @@ export const LEGO_COLORS = [
   [137, 135, 136],
 ];
 
-export const closestMatch = (rgb) => {
+
+export const closestMatch = (rgb, colorOptions) => {
+  // console.log(colorOptions);
+  const COLORS = [];
+  for (let i = 0; i < colorOptions.length; i += 1) {
+    // console.log(COLORS);
+    COLORS.push(LEGO_COLORS[colorOptions[i]]);
+  }
```

Comment:
Since you're just mapping colorOptions onto color values, it's clearer to change it to this:

```suggestion
  const COLORS = colorOptions.map(colorIndex => LEGO_COLORS[colorIndex]);
```